src/source/amplify.rs

```diff
@@ -12,6 +12,11 @@ where
     Amplify { input, factor }
 }
 
+/// converts decibels to linear
+pub fn to_linear(decibels: f32) -> f32 {
```

Comment:
why is this public? (its generally good to keep as much private as possible that way its not breaking if we remove/change it in the future)